addons/account/views/account_payment_term_views.xml

```diff
@@ -105,14 +105,10 @@
             <field name="model">account.payment.term</field>
             <field name="arch" type="xml">
                 <kanban class="o_kanban_mobile">
-                    <field name="name"/>
-                    <field name="note"/>
                     <templates>
-                        <t t-name="kanban-box">
-                            <div t-attf-class="oe_kanban_global_click">
-                                <div><strong class="o_kanban_record_title"><t t-out="record.name.value"/></strong></div>
-                                <div t-if="!widget.isHtmlEmpty(record.note.raw_value)"><t t-out="record.note.value"/></div>
-                            </div>
+                        <t t-name="kanban-card">
+                            <field class="fw-bold fs-5" name="name"/>
```

Comment:
same thing here, why not `fw-bolder`?
for comparison this is is prior to your commit 
![image](https://github.com/odoo/odoo/assets/46533567/cf405517-88d3-49a8-91de-dc2f182be540)
and this is post
![image](https://github.com/odoo/odoo/assets/46533567/6efa8766-a9dd-40d8-b4d2-dfe4f13a5261)

personally I think `fw-bolder` would look better in this scenario